modules/devguide/examples/Cloud.java

```diff
@@ -42,7 +42,7 @@
 
 /**
  * Example of Cas (Check and Set) handling in Java for the Couchbase Developer
- * Guide ported from 2.x. See StartUsingCloud.java for 3.x.
+ * Guide ported from 2.x. See StartUsingCapella.java for 3.x.
  */
 
 public class Cloud {
```

Comment:
Why is this class alone still called Cloud and not Capella :)

Reply:
And the file name ;)

Reply:
Hi Arun! I don't know. I didn't want to touch it, because it wasn't related to the change at hand... and I needed to resist the temptation to fix everything at once ;-)

Reply:
No that makes perfect sense. the only other recommendation I had was you mentioned `address` to be precise probably call it hostaddress because thats what it is ?

Reply:
Resolving this conversation, since there's no longer a separate variable for the address.